raw_description/urdf/base_mini/base.urdf.xacro

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find cob_description)/urdf/common.xacro"/>
+  <xacro:include filename="$(find raw_description)/urdf/base_mini/base.gazebo.xacro" />
+
+  <!-- drive_wheel module -->
+  <xacro:include filename="$(find raw_description)/urdf/mecanum_wheel/mecanum_wheel.urdf.xacro" />
+
+
+  <!-- properties -->
+  <xacro:property name="caster_offset_x" value="0.115" />
+  <xacro:property name="caster_offset_y" value="0.09" />
+  <xacro:property name="caster_offset_z" value="0.00" />
+
+  <xacro:property name="battery_offset_x" value="0.0" />
+  <xacro:property name="battery_offset_y" value="-0.008" />
+  <xacro:property name="battery_offset_z" value="0.065" />
+
+  <xacro:property name="base_mass" value="43.1" />
+  <xacro:property name="battery_mass" value="40" />
+
+  <xacro:property name="battery_size_x" value="0.5" />
+  <xacro:property name="battery_size_y" value="0.148" />
+  <xacro:property name="battery_size_z" value="0.26" />
+
+
+  <xacro:macro name="raw_base" params="name">
+
+    <link name="${name}_footprint"/>
+
+    <joint name="${name}_footprint_joint" type="fixed">
+      <origin xyz="0 0 0.05" rpy="0 0 0" />
+      <parent link="${name}_footprint"/>
+      <child link="${name}_link" />
+    </joint>
+
+    <link name="${name}_link">
+      <inertial>
+        <origin xyz="0 0 0" />
+        <mass value="${base_mass}" />
+        <inertia ixx="9.64" ixy="-0.02" ixz="0.44" iyy="11.33" iyz="-0.007" izz="3.07" />
+      </inertial>
```

Comment:
where does this come from?
maybe you want to use `box_inertial` from `common.xacro`

Reply:
I couldn't find a reliable source so i guestimated the weight and used the box_inertial